src/components/collapsible/Collapsible.tsx

```diff
@@ -102,6 +102,7 @@ class Collapsible extends React.PureComponent<CollapsibleProps, CollapsibleState
         const resinTagTarget: string = RESIN_TAG_TARGET;
         const modifiedButtonProps: { [index: string]: string } = omit(buttonProps, [resinTagTarget]);
         const interactionTarget = buttonProps[resinTagTarget];
+        const ariaExpanded = isOpen ? 'true' : 'false';
```

Comment:
Can we in-line this code with its prop since it isn't reused?

Reply:
Will do! :D

Reply:
Done!